index.html

```diff
@@ -629,7 +629,101 @@ <h3>Presentations</h3>
+repository engineered to store verifiable credentials, and then the
+presentation to a verifer. In <a href="#ex-1-example-verifiable-credential">
+Example 1</a> Pat receives an "AlumniCredential" from "Example University" that 
+will be stored in Pat's digital wallet on his mobile device, a type of 
+credential repository.
+      </p>
+
+      <pre class="example nohighlight" title="Example Verifiable Credential">
+{
+  "@context": [
+    "https://w3.org/2018/credentials/v1",
+    "https://example.com/examples/v1"
+  ],
+  "id": "http://example.edu/credentials/1872",
+  "type": ["VerifiableCredential", "AlumniCredential"],
+  "issuer": "https://example.edu/issuers/565049",,
+  "issuanceDate": "2010-01-01T19:73:24Z",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "alumniOf": "Example University"
+  },
+  "proof": {
+    "type": "RsaSignature2018",
+    "created": "2017-06-18T21:19:10Z",
+    "creator": "https://example.com/jdoe/keys/1",
+    "nonce": "c0ae1c8e-c7e7-469f-b252-86e6a0e7387e",
+    "signatureValue": "BavEll0/I1zpYw8XNi1bgVg/sCneO4Jugez8RwDg/+
+      MCRVpjOboDoe4SxxKjkCOvKiCHGDvc4krqi6Z1n0UfqzxGfmatCuFibcC1wps
+      PRdW+gGsutPTLzvueMWmFhwYmfIFpbBu95t501+rSLHIEuujM/+PXr9Cky6Ed
+      +W3JT24="
+  }
+}
+      </pre>
+
+      <p>
+In the final chapter of this lifecycle, Pat's "AlumniCredential" will be 
+presented for a discount. The verifier, a ticket sales counter, states that any
+alumni of "Example University" will receive a 72% discount on football season 
+tickets. Pat uses thier mobile device to select the Verifiable Credential 
+received earlier to send to the verifier in the form of a Verifiable 
+Presentation in order to receive the discount, this can be found
+in <a href="#ex-2-example-verifiable-presentation">Example 2</a>.
+      </p>
+      <pre class="example nohighlight" title="Example Verifiable Presentation">
+{
+  "@context": [
+    "https://w3.org/2018/credentials/v1",
+    "https://example.com/examples/v1"
+  ],
+  "id": "did:example:Fg15hCqJ19LP4X4RaiumJVb8YeZnzUjWLh74hKqBvk5B",
+  "type": "VerifiablePresentation",
+  "verifiableCredential": [{
+    "id": "http://example.edu/credentials/1872",
+    "type": ["VerifiableCredential", "AlumniCredential"],
+    "issuer": "https://example.edu/issuers/565049",,
+    "issuanceDate": "2010-01-01T19:73:24Z",
+    "credentialSubject": {
+      "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+      "alumniOf": "Example University"
+    },
+    "proof": {
+      "type": "RsaSignature2018",
+      "created": "2017-06-18T21:19:10Z",
+      "creator": "https://example.com/jdoe/keys/1",
+      "nonce": "c0ae1c8e-c7e7-469f-b252-86e6a0e7387e",
+      "signatureValue": "BavEll0/I1zpYw8XNi1bgVg/sCneO4Jugez8RwDg/+
+      MCRVpjOboDoe4SxxKjkCOvKiCHGDvc4krqi6Z1n0UfqzxGfmatCuFibcC1wps
+      PRdW+gGsutPTLzvueMWmFhwYmfIFpbBu95t501+rSLHIEuujM/+PXr9Cky6Ed
+      +W3JT24="
+    }
+  }],
+  "proof": { ... }
+}
```

Comment:
As this is the only (is there another one?) example of a complete presentation in the data model, could we also have an example of a presentation done with selective disclosure?

Reply:
We may want to compare/contrast between a credential that has certain attributes selectively disclosed vs. one that has not. Since these examples take up a lot of space, I suggest putting it in the appendix like we did for the JWT / LDP compare-contrast.

Reply:
@brentzundel I see no reason why we couldn't have another example. My thinking is to get this PR in and then make another one that would add to this section.

@msporny Are you suggesting to add a selective disclosure example to this section and then complete the compare-contrast in the appendix? If so, I agree with that line of thinking.

Reply:
Sounds good to me.